bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -148,7 +148,7 @@
 
 		<label>UniFi PoE Port</label>
 		<description>A Power Over Ethernet (PoE) port on a UniFi switch</description>
-
+		<semantic-equipment-tag>NetworkSwitch</semantic-equipment-tag>
```

Comment:
This is a single port on a network switch, I'm in doubt whether this qualifies for the tag. You could end up seeing many of these.

Reply:
Ditto above.